briefin/src/app/globals.css

```
@@ -0,0 +1,169 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@100..900&display=swap');
```

Stylelint (17.4.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@100..900&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@100..900&display=swap'" (import-notation)

(import-notation)